docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+
+Wait patiently as your fork is being created.
+
+When the forking process is complete, the repository will show up as `{your user id}/loopback-next`. In my case, it is `emonddr/loopback-next` .
+
+![submit_pr_my_forked_repo.png](./imgs/submit_pr_my_forked_repo.png)
+
+
+
+### 3. Create a feature branch for your work
+
+Notice your repo has a `master` branch already created ( refer bottom left corner of the picture above). It is commonplace to have this branch represent the latest, clean version of this repository's content. For the purposes of your PR, let's create a feature branch with a name indicative of your changes. In my case, it is `emonddr-doc-changes`.
+
+Click on the `down arrow` on the right-hand side of the `master` branch button.
+
+![submit_pr_create_feature_branch_1.png](./imgs/submit_pr_create_feature_branch_1.png)
+
+This brings up a small dialog for finding or creating a branch.
+
+![submit_pr_create_feature_branch_2.png](./imgs/submit_pr_create_feature_branch_2.png)
+
+Enter the `name` of your new `feature branch` in the search field. Because the branch `emonddr-doc-changes` doesn't yet exist, a menu item to create it appears below the search field. Click on the `Create branch` link.
+
+![submit_pr_create_feature_branch_3.png](./imgs/submit_pr_create_feature_branch_3.png)
+
+Your feature branch is created.
+
+![submit_pr_create_feature_branch_4.png](./imgs/submit_pr_create_feature_branch_4.png)
+
+
+### 4. Clone a local copy of your repository
+
+Click on the `Clone or download` button
+
+![submit_pr_clone_1.png](./imgs/submit_pr_clone_1.png)
+
+This brings up tiny dialog with different choices : Clone with SSH, Use HTTPS, Open in Desktop, or Download Zip).
+
+In my case, I will leave it as `Clone with SSH`, and click on the `copy to clipboard` icon on the right side of the repository url (we are going to paste this value in a terminal window soon).
+
+![submit_pr_clone_2.png](./imgs/submit_pr_clone_2.png)
+
+
+Open a terminal window, and navigate to the directory where you want to clone the repository. In my case, this is `/Users/dremond/git` .
+
+(For all the terminal commands mentioned below, be sure to press the `ENTER` key.)
+
+![submit_pr_clone_3.png](./imgs/submit_pr_clone_3.png)
+
+Type `git clone ` and `Ctrl+V` to paste the github repository url currently in memory.
+
+You should have
+
+```
+git clone git@github.com:{your user id}/{repository name}.git
+```
+
+In my case it is
+
+```
+git clone git@github.com:emonddr/loopback-next.git
+```
+Wait until the cloning process completes.
+
+![submit_pr_clone_4.png](./imgs/submit_pr_clone_4.png)
+
+### 5. Navigate to your repository directory
+
+Type
+
+```
+cd loopback-next
+```
+
+![submit_pr_clone_5.png](./imgs/submit_pr_clone_5.png)
+
+Type
+
+```
+git status
+```
+
+and you will see that you are currently on the `master` branch.
+
+![submit_pr_clone_6.png](./imgs/submit_pr_clone_6.png)
+
+You can also see this by typing
+
+```
+git branch -a
```

Comment:
Maybe `git branch` by itself if all they want to see is that they're on the `master` branch?

Reply:
I think we should go directly to `git checkout emonddr-doc-changes`. The fewer steps, the better.